Source/WebCore/loader/cache/AllowedFonts.cpp

```diff
@@ -851,16 +852,19 @@ bool isFontBinaryAllowed(const void* data, size_t size, DownloadableBinaryFontAl
         return false;
     case DownloadableBinaryFontAllowedTypes::Restricted: {
         auto sha = hashForFontData(data, size);
-        return allowedFontHashesInLockdownMode().contains(sha);
+        auto allowedFontHashes = allowedFontHashesInLockdownMode().contains(sha);
+        if (!allowedFontHashes)
+            RELEASE_LOG(Fonts, "[Lockdown Mode] This font is blocked: %s", url.utf8().data());
```

Comment:
As far as I know, we don't allow logging URLs. It may be acceptable if you use:
`RELEASE_LOG(Fonts, "[Lockdown Mode] This font is blocked: %" SENSITIVE_LOG_STRING, url.utf8().data());`

Note that it will then not show in sysdiagnoses and you'll need enable sensitive logging on your machine when you want to see this logging. Otherwise, if you want to log in sysdiagnoses, I recommend logging something else than the URL (maybe font name?).

Reply:
Ah! Good point. hmmm… 
And this is indeed the requested URL from the CSS. 

Probably we do not have access to the real font name at this stage (as the download is blocked because of lockdown mode), probably the parsed name from the CSS. 

Hmm… How do I access this?
But could this create also issues by revealing the name of the fonts in the log?

@brentfulgham what would be the more helpful choice, with regards to what you experienced in the past for Lockdown mode bugs?

Reply:
I think just knowing that we blocked a font because of LDM is helpful. The best case would be to know the name of the font so we could add it to the set of fonts we review and approve.

However, since it will often be a potentially randomized data URL that may not be possible.

So for now, I think just noting the event is useful.

Reply:
done.